### PR TITLE
Flyway

### PR DESCRIPTION
**Pull request by MrK191**

### CODE REVIEW

File: src/main/resources/META-INF/persistence.xml [hunk 1]
```diff
@@ -11,7 +11,7 @@
 	    <!-- comment out the line below to make the app NOT build 
 		 persistence tables on startup; will make deployment
 		 faster. - L.A. -->
-            <property name="eclipselink.ddl-generation" value="create-tables"/>
+            <property name="eclipselink.ddl-generation" value="none"/>
```
Comment (madryk) [inline src/main/resources/META-INF/persistence.xml:12]: Above comment lost it's original meaning. I would remove it.

File: src/main/resources/db/migration/V1__baseSchema.sql [hunk 2]
```diff
@@ -0,0 +1,2072 @@
+create table workflow
+(
+	id serial not null
+		constraint workflow_pkey
+			primary key,
+	name varchar(255)
+);
+
+alter table workflow owner to dvnapp;
```
Comment (madryk) [inline src/main/resources/db/migration/V1__baseSchema.sql]: I don't see much sense in changing ownership to tables in creation of tables process.

We're hardcoding db username in this script.

In my opinion we should get rid of it.

It should not generate any problems. And if it does then setting ownership on the whole db
should fix it.

File: pom.xml [hunk 3]
```diff
@@ -718,7 +724,7 @@
             <plugin>
                 <!-- https://stackoverflow.com/questions/46177921/how-to-run-unit-tests-in-excludedgroups-in-maven -->
                 <artifactId>maven-surefire-plugin</artifactId>
-                <version>2.22.0</version>
+                <version>3.0.0-M1</version>
```
Comment (madryk) [inline pom.xml:727]: Any particular reason for using higher plugin version?

M1 suffix points that this version could be not so stable as `2.22.x`

**Review by madryk — CHANGES_REQUESTED** [review]

File: pom.xml (hunk 3 above)
Comment (MrK191) [reply pom.xml:727]: It was due to a bug in OpenJdk8 debian distro + a bug in surefire plugin. I was unable to complete maven clean install due to build failing. I found on stackoverflow that it was fixed in M1 version. Another solution would be to change Java to Oracle distro which I'm using atm, but anyone using OpenJdk on debian could run into this problem.

File: src/main/resources/db/migration/V1__baseSchema.sql (hunk 2 above)
Comment (MrK191) [reply src/main/resources/db/migration/V1__baseSchema.sql]: Ok i will remove it, it was generated when i was dumping schema.

File: pom.xml (hunk 3 above)
Comment (madryk) [reply pom.xml:727]: Ok. If earlier version is making problems then we can leave `3.0.0-M1` version

**Review by madryk — APPROVED** [review]

**Pull request merged**